testing/src/test/java/io/stargate/it/http/graphfirst/AdminResourcesTest.java

```diff
@@ -0,0 +1,64 @@
+package io.stargate.it.http.graphfirst;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import com.apollographql.apollo.ApolloClient;
+import com.example.graphql.client.schema.GetTableQuery;
+import com.example.graphql.client.schema.type.BasicType;
+import com.example.graphql.client.schema.type.ColumnInput;
+import com.example.graphql.client.schema.type.DataTypeInput;
+import io.stargate.it.http.graphql.GraphqlITBase;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+import java.util.concurrent.ExecutionException;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+public class AdminResourcesTest extends GraphqlITBase {
+  private static String KEYSPACE;
+
+  @BeforeAll
+  public static void crateTestKeyspace() {
+    KEYSPACE = UUID.randomUUID().toString();
+    session.execute(
+        "CREATE KEYSPACE IF NOT EXISTS ks_1 WITH replication = {'class': 'SimpleStrategy', 'replication_factor': 1 };");
+  }
+
+  @Test
+  public void createTable() throws ExecutionException, InterruptedException {
```

Comment:
I am trying to write an IT test for admin resources to understand how schema retrieval works. I am trying to base it on the exiting cql-first `GraphqlTest`. 
From what I see, this test is pre-creating all schema via cql query from the `betterbotz.cql`. 
Regarding the new `graph-first` API my impression is that we should allow end-user to create everything via `graph-ql` queries only. Is that true?
If yes, can the new test use the same types, i.e: ` GetTableQuery.Table` or do we need to define new ones?

Reply:
`GraphqlTest` uses the Apollo client, this is good to have an example with a "real-life" GraphQL library, but adding new tests requires a lot of boilerplate. See the discussion on #270.
For our schema-first tests I think it will be faster to use the approach in `GraphqlJWTAuthTest`.

> Regarding the new `graph-first` API my impression is that we should allow end-user to create everything via `graph-ql` queries only. Is that true?

Yes. I think most of the time we won't need to run any CQL queries, the test will start with a `deploySchema` operation, and proceed from there. Except of course to cover specific database states (e.g. migration of an existing schema, `graphql_schema` exists but with the wrong columns, etc).